app/locale/ar.coffee

```diff
@@ -951,10 +951,6 @@ module.exports = nativeDescription: "العربية", englishDescription: "Arabi
 #    maka_title: "Customer Advocate"
 #    robin_title: "Senior Product Manager"
 #    nolan_title: "Sales Manager"
-#    lisa_title: "Business Development Manager"
-#    sean_title: "Territory Manager"
-#    liz_title: "Territory Manager"
-#    jane_title: "Customer Success Manager"
```

Comment:
Here we'll need a commented out stub for the

```
titles:
  csm:
  territory_manager:
```

We have a script that can make these changes (but it also leaves large artifacts as well).
It may be easier to manually tidy up.

Reply:
I've added the commented out lines manually here.  Was going to update the script and then run it to propagate these changes but I think it's best to separate the two.  We should have what we need now.